src/DMS/Service/Meetup/Response/MultiResultResponse.php

```diff
@@ -59,9 +61,23 @@ public function filter(Closure $p)
         $clone = clone $this;
 
         $clone->setData(array_filter($this->data, $p));
+        $clone->updateBody();
 
         return $clone;
     }
+
+    /**
+     * Assigns a new body to the request, based on the data it contains.
+     */
+    private function updateBody()
+    {
+        $data = array('meta' => $this->metadata, 'results' => $this->data);
+
+        $this->body = EntityBody::factory(
+            $this->isContentType('json') ? json_encode($data) : http_build_query($data)
```

Comment:
This check was just done because we have the same thing on `parseBodyContent()`. Maybe would be nice to just use `json` and remove this extra complexity.